core/templates/pages/topics-and-skills-dashboard-page/create-new-skill-modal.controller.ts

```diff
@@ -57,14 +61,21 @@ angular.module('oppia').controller('CreateNewSkillModalController', [
       $scope.conceptCardExplanationEditorIsShown = true;
     };
 
-    $scope.updateSkillDescription = function() {
+    $scope._SkillEditorStateServiceCallback = function() {
+      $scope.skillDescriptionExists = (
+        SkillEditorStateService.getSkillDescriptionExists());
+    };
+    $scope.updateSkillDescriptionAndCheckIfExists = function() {
```

Comment:
Hmmm maybe I'm not sure what `SkillEditorStateService.markChangeInSkillDescription()` does. I thought it marks a change so that the frontend will re-render. When I named this function as `updateSkillDescriptionAndCheckIfExists`, I didn't mean it chronologically. The function just performs both of those actions. Should I change the logic here?

Reply:
Done